Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -662,16 +670,16 @@ def merge(args:)
 
               all_bottle_hash = { formula_name => all_bottle_formula_hash }
 
-              puts "Copying #{local_filename} to #{all_local_filename}" if args.verbose?
-              FileUtils.cp local_filename, all_local_filename
+              puts "Copying #{filename} to #{all_filename}" if args.verbose?
+              FileUtils.cp filename.to_s, all_filename.to_s
 
-              all_local_json_path = Pathname(all_local_json_filename)
+              all_local_json_path = Pathname(filename.json)
```

Comment:
`all_filename.json`?

Reply:
🤦🏻 . Fixed (by you!) in https://github.com/Homebrew/brew/pull/11274